docs/reference/cluster/stats.asciidoc

```diff
@@ -231,6 +231,18 @@ Total size of all shards assigned to selected nodes.
 `size_in_bytes`::
 (integer)
 Total size, in bytes, of all shards assigned to selected nodes.
+
+`reserved`::
+(<<byte-units,byte value>>)
+A prediction of how much larger the shard stores on this node will eventually
```

Comment:
Thanks, yes. Actually not quite cluster-wide since you can select a subset of nodes here but "these nodes" is awkward so I removed it in https://github.com/elastic/elasticsearch/pull/58029/commits/b89a56770caa137261dd7f5e03336b7b22af2012.